src/rely/Rely.re

```diff
@@ -31,6 +31,7 @@ module Describe = {
   type describeConfig = {
```

Comment:
this type is going to be extracted into an argument for a functor for creating TestSuiteRunners come combining of TestFrameworks/separation of discovery and execution. Otherwise I would probably rename or make getTime an argument.